### ISSUE TITLE
Show names of all contributors not only last one

### ISSUE THREAD

**Issue by Briseur** [KapiX/Polyglot#46]
Sometimes, you come to a translation only to fix a typo. It's a contribution like another but now your name replaced the one of the person who did 99% of the job on Contributors list.
Perhaps renaming that list to Recent Contributors would be better.
What could be done is to list on a page all contributors for a specific language. It could be on per language progress page that I suggested on another ticket.

Also if the file mention translators, it should say 'Translated (to ...) by Polyglot community' to avoid the problem. Though that may not always work for existent apps, it could be an idea of  generic line for new ones.

**Comment by KapiX** [comment]
That's not how "Contributors" panel works. It doesn't just delete people if they are no longer an author of any existing translation. If you contributed even a small fix to an app only once, you stay there forever.
Did you notice a different behavior? With which app?

**Comment by Briseur** [comment]
Indeed, after checking few apps, I finally found some with different contributors for French. I guess I didn't notice because they are not sorted by language and I did few alone.
